images/base-emu/build/Dockerfile

```diff
@@ -91,6 +91,9 @@ github_download "cemu-project/Cemu" "cemu-emu.AppImage" "" "/latest"
 echo "**** Downloading Dolphin AppImage ****"
 github_download "pkgforge-dev/Dolphin-emu-AppImage" "dolphin-emu.AppImage" "|select(.name|contains(\"dwarfs-x86_64\"))"
 
+echo "**** Downloading Citron Emulator AppImage ****"
+github_download "pkgforge-dev/Citron-AppImage" "citron.AppImage" "|select(.name|contains(\"anylinux-x86_64_v3\"))"
```

Comment:
Is there a reason why you are pulling from the unofficial https://github.com/pkgforge-dev/Citron-AppImage instead of getting the latest releases directly from the official git repo https://git.citron-emu.org/citron/emu/-/releases ?  

Or is it just to re-use the `github_download` script?